<?xml version="1.0"?>

<robot name="qube" xmlns:xacro="http://www.ros.org/wiki/xacro">
  <xacro:include filename="$(find qube_description)/urdf/qube.macro.xacro"/>

  <xacro:property name="box_width" value="0.102"/>
  <xacro:property name="angle_link_length" value="0.025"/>
  <xacro:property name="angle_link_width" value="0.002"/>

  <link name="world"/>
  <link name="base_link"/>

  <joint name="base_joint" type="fixed">
    <parent link="world"/>
    <child link="base_link"/>
    <origin xyz="0 0 0" rpy="0 0 0"/>
  </joint>

  <xacro:joint_model prefix=""/>

</robot>